<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>274</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>444</width>
    <height>274</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>444</width>
    <height>274</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>58</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/img/Icons/user2.svg);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>58</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/img/Icons/lock.svg);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_enter">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>90</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ENTRAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>SAIR</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_name">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Digite seu nome:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_passwd">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Digite sua senha:</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;AUTOMATIZADOR DE E-MAILS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>229</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_register">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cadastrar</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
